.gitignore

```
@@ -5,13 +5,13 @@
 *.d
 *.log
 dump.rdb
-placeholderkv-benchmark
-placeholderkv-check-aof
-placeholderkv-check-rdb
-placeholderkv-check-dump
-placeholderkv-cli
-placeholderkv-sentinel
-placeholderkv-server
+valkey-benchmark
+valkey-check-aof
+valkey-check-rdb
+valkey-check-dump
+valkey-cli
+valkey-sentinel
+valkey-server
```

Cover both sets of filenames/symlinks?

```suggestion
*-benchmark
*-check-aof
*-check-rdb
*-check-dump
*-cli
*-sentinel
*-server
```

---

This is a good idea!